### PR TITLE
[New Rule] SSH Authorized Keys File Modification

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/753

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by ansell** [comment]
Personally, I would rather if the noisy processes were setup on a case-by-case basis as exceptions for each system by each installations maintainer because this rule is critical to system authentication and access auditing.

**Comment by Samirbous** [comment]
> Personally, I would rather if the noisy processes were setup on a case-by-case basis as exceptions for each system by each installations maintainer because this rule is critical to system authentication and access auditing.

@ansell agree, all file changes are logged and stored, this rule is more suited for alerting abnormal changes (thus the exclusion of some noisy processes otherwise it will flood the dashboard), of course for compliance or hunting needs (e.g. reporting) one can duplicate the same query with no filtering of noisy processes such as use of vim to edit interactively the file.

File: rules/cross-platform/persistence_ssh_authorized_keys_modification.toml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/12/22"
+maturity = "production"
+updated_date = "2020/12/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+The SSH Authorized keys file specifies which users are allowed to log into a server using public key authentication.
+Adversaries may modify it to maintain persistence on a victim host by adding their own public key(s).
```
Comment (bm11100) [inline rules/cross-platform/persistence_ssh_authorized_keys_modification.toml:10]:
```suggestion
The Secure Shell (SSH) authorized_keys file specifies which users are allowed to log into a server using public key authentication.
Adversaries may modify it to maintain persistence on a victim host by adding their own public key(s).
```

Comment (bm11100) [reply rules/cross-platform/persistence_ssh_authorized_keys_modification.toml:10]: may need a re lint

**Review by bm11100 — APPROVED** [review]
lgtm 😊

**Pull request merged**